tests/std/tests/P1007R3_assume_aligned/test.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <memory>
+
+static void can_change_value_via_assume_aligned(int* p) {
+    int* p1 = std::assume_aligned<256>(p);
+    p1[0]   = 10;
+    p1[1]   = -15;
+}
+
+static constexpr int can_read_constexpr_value_via_assume_aligned_and_sum_it(int const* p, size_t size) {

```

Comment:
Technically need to include `<stddef.h>` for `size_t`, or `<cstddef>` for `std::size_t` (if you want to qualify the occurrences).